tensorboard/plugins/debugger/events_writer_manager.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Manages writing debugger-related events to disk.
+
+Creates new events files if previous files get too big.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import re
+import threading
+import time
+
+import tensorflow as tf
+
+
+# Files containing debugger-related events should start with this string.
+DEBUGGER_EVENTS_FILE_STARTING_TEXT = "events.debugger"
+
+# A regex used to match the names of debugger-related events files.
+_DEBUGGER_EVENTS_FILE_NAME_REGEX = re.compile(
+    "^" + re.escape(DEBUGGER_EVENTS_FILE_STARTING_TEXT) + "\\.\\d+\\.(\\d+)")
```

Comment:
Please replace the last string with the raw string literal `r'\.\d+\.(\d+\)`, and also the first one with `r'^'` for consistency.